stats/stats.go

```diff
@@ -59,6 +59,27 @@ func (s *Begin) IsClient() bool { return s.Client }
 
 func (s *Begin) isRPCStats() {}
 
+// InitialResolverResult represents an event that the resolved finished
+// resolving if the ClientConn blocks on resolver resolution while performing a
+// RPC.
+type InitialResolverResult struct{}
+
+// IsClient indicates if the stats information is from client side. Only Client
+// Side interfaces with a resolver, thus always returns true.
+func (irr *InitialResolverResult) IsClient() bool { return true }
+
+func (irr *InitialResolverResult) isRPCStats() {}
+
+// PickerUpdated represents an event that the picker finished picking if the
+// ClientConn blocks on picker picking while performing a RPC.
```

Comment:
"PickerUpdated indicates that the LB Policy's picker was updated after the previous pick attempt returned ErrNoSubConnAvailable."  Or something?

Reply:
I don't think this is the only case that this gets triggered (see list of 4 in https://github.com/grpc/grpc-go/blob/master/picker_wrapper.go#L87). Any other ideas?

Reply:
Yeah, that's true.  I was trying to keep it simple.  The alternative would be something more vague like "PickerUpdated indicates that the LB policy provided a new picker while the RPC was waiting for one."

Reply:
Let's do the more vague one?

Reply:
Sure..  "picker finished picking if the CC blocks on picker picking" makes no sense to me 😛

Reply:
Please update to something like `PickerUpdated indicates that the LB policy provided a new picker while the RPC was waiting for one.`